app/models/manageiq/providers/redhat/discovery.rb

```diff
@@ -0,0 +1,20 @@
+require 'manageiq/network/port'
+require 'ovirt'
+
+module ManageIQ
+  module Providers
+    module Redhat
+      class Discovery
+        def self.probe(ost)
+          Ovirt.logger = $rhevm_log if $rhevm_log
+
+          if Port.open?(ost, Ovirt::Service::DEFAULT_PORT)
```

Comment:
I think this needs to be fully qualified as `ManageIQ::Network::Port`.

Reply:
Done.